weeb/module.py

```diff
@@ -0,0 +1,145 @@
+import discord
+from discord.ext import commands
+from discord.abc import PrivateChannel
+import aiohttp
+import asyncio
+from core import utils, text, logging, i18n
+
+_ = i18n.Translator("modules/fun").translate
+
+
+class Weeb(commands.Cog):
+    """
+    Weeby commands for your own Weebiness
+
+    Includes:
+        ?sauce that will give your desired sauce if you supply it with a magic number
+    """
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @classmethod
+    def doujin_tags(cls, dic):
+        characters, parodies, tags, artists, groups, languages, categories = (
+            [],
+            [],
+            [],
+            [],
+            [],
+            [],
+            [],)
+        tags = []
+        for tag in dic["tags"]:
+            if tag["type"] == "character":
+                characters.append(tag["name"])
+            elif tag["type"] == "parody":
+                parodies.append(tag["name"])
+            elif tag["type"] == "tag":
+                tags.append(tag["name"])
+            elif tag["type"] == "artist":
+                artists.append(tag["name"])
+            elif tag["type"] == "group":
+                groups.append(tag["name"])
+            elif tag["type"] == "language":
+                languages.append(tag["name"])
+            elif tag["type"] == "category":
+                categories.append(tag["name"])
+
+        characters = ["Characters", ",".join(str(e) for e in characters)]
+        parodies = ["Parodies", ",".join(str(e) for e in parodies)]
+        tags = ["Tags", ",".join(str(e) for e in tags)]
+        artists = ["Artists", ",".join(str(e) for e in artists)]
+        groups = ["Groups", ",".join(str(e) for e in groups)]
+        languages = ["Languages", ",".join(str(e) for e in languages)]
+        categories = ["Categories", ",".join(str(e) for e in categories)]
+
+        tags = [
+            characters,
+            parodies,
+            tags,
+            artists,
+            groups,
+            languages,
+            categories
+        ]
+        return tags
+
+    def embed(self, ctx, dic, DOUJIN_ID):
+        tags = self.doujin_tags(dic)
+
+        url = f"https://nhentai.net/g/{DOUJIN_ID}/".format(DOUJIN_ID=DOUJIN_ID)
```

Comment:
You do not need to use .format() when using f-strings, it should work just as `f"https://nhentai.net/g/{DOUJIN_ID}/"`